tests/testthat/test_pipelines.R

```diff
@@ -135,3 +135,94 @@ test_that("survtoclassif_IPCW", {
   # different cutoff times, different (crank) predictions
   expect_false(all(p$crank == p2$crank))
 })
+
+test_that("survtoregr_PEM", {
+  skip_if_not_installed("mlr3learners")
+  require_namespaces('glmnet')
+    
+  
+  task = tsk('rats')
+  # for this section, select only numeric covariates, 
+  # as 'regr.glmnet' does not automatically handle factor type variables
+  task$select(c('litter', 'rx'))
+  learner = lrn('regr.glmnet', 
+                family = "poisson", 
+                lambda = 0,
+                use_pred_offset = FALSE)
+  
+
+  pipe = ppl("survtoregr_PEM", learner = learner)
+  expect_class(pipe, "Graph")
+  grlrn = ppl("survtoregr_PEM", learner = learner, graph_learner = TRUE)
+  expect_class(grlrn, "GraphLearner")
+  expect_equal(grlrn$predict_type, "response")
+  
+  grlrn$train(task)
+  p = grlrn$predict(task)
+  expect_prediction_surv(p)
+  
+  
+  cox = lrn("surv.coxph")
+  suppressWarnings(cox$train(task))
+  p2 = cox$predict(task)
+  
+  expect_equal(p$row_ids, p2$row_ids)
+  expect_equal(p$truth, p2$truth)
+  expect_equal(p$score(), p2$score(), tolerance = 0.015) 
+  
+  # Test with cut
+  grlrn = ppl("survtoregr_PEM", 
+              learner = learner,
+              cut = c(10, 30, 50),
+              graph_learner = TRUE)
+  expect_class(grlrn, "GraphLearner")
+  suppressWarnings(grlrn$train(task))
+  p = grlrn$predict(task)
+  expect_prediction_surv(p)
```

Comment:
check that ncol(p$data$distr) == 3? and exaclty the cut points? (if I recall correctly that's the time points used)